src/webserver.rs

```diff
@@ -451,8 +450,7 @@ async fn handle_validate_hashchain_proof(
     // debug!("Verifying zkSNARK proof...");
     match groth16::verify_proof(&pvk, &proof, &[public_param]) {
         Ok(_) => {
-            println!("Proof is valid");
-            println!("Verified with: {:?}", public_param);
+            info!("Proof is valid\nVerified with: {:?}", public_param);
```

Comment:
Multiline logs end up getting formatted weird and will be difficult later when doing log parsing, could we maybe make it into one?

info!("proof successfully verified with: {:?}", public_param);